app/routes.js

```diff
@@ -40,33 +40,43 @@ router.get('/solutions', (req, res) => {
   res.render('pages/browse-solutions/template.njk', addConfig(context));
 });
 
-router.get('/solutions/capabilities-selector', withCatch(async (req, res) => {
-  logger.info('navigating to capabilities-selector page');
-  const context = await getCapabilitiesContext();
-  res.render('pages/capabilities-selector/template.njk', addConfig(context));
+router.post('/solutions/capabilities-selector', withCatch(async (req, res) => {
+  let capabilitiesSelected = req.body.capabilities;
+  if (!Array.isArray(capabilitiesSelected)) capabilitiesSelected = [capabilitiesSelected];
+  const capabilitiesParam = capabilitiesSelected.join('+');
+  res.redirect(`/solutions/capabilities-selector.${capabilitiesParam}`);
 }));
 
-router.get('/solutions/:filterType', withCatch(async (req, res) => {
-  const { filterType } = req.params;
-  logger.info(`filter type '${filterType}' applied`);
-  const context = await getSolutionListPageContext({ filterType });
-  res.render('pages/solutions-list/template.njk', addConfig(context));
-}));
-
-router.post('/solutions/custom', withCatch(async (req, res) => {
-  const capabilitiesSelected = req.body.capabilities;
-  const context = await getSolutionsForSelectedCapabilities({ capabilitiesSelected });
-  res.render('pages/solutions-list/template.njk', addConfig(context));
+router.get('/solutions/:filterType.:capabilities?', withCatch(async (req, res) => {
+  const { filterType, capabilities } = req.params;
+  if (filterType === 'capabilities-selector') {
+    if (capabilities) {
+      const formattedCapabilities = capabilities.split('+');
+      const context = await getSolutionsForSelectedCapabilities({
+        capabilitiesSelected: formattedCapabilities,
+      });
+      logger.info(`navigating to capabilities-selector (with ${capabilities} selected) page`);
```

Comment:
Should this be navigating to the solutions-list page?

Reply:
yes